app/src/main/AndroidManifest.xml

```diff
@@ -11,8 +11,6 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
 
     <application
-        android:allowBackup="true"
```

Comment:
Leaving this in here will cause conflicts when users of the library also define this, so delete this.